.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +1 @@
-blank_issues_enabled: false
+blank_issues_enabled: true
```

Comment:
Please get rid of both this blank issues change and the normal issue.yaml files, they are unrelated to this PR and allowing the users to enter whatever they want in issues, just leads to spam in a repo as large as FreeTube.

Reply:
Apologies! We were using this for our own documentation. Thanks for reminding us.